### ISSUE TITLE
Hitting enter on creator search box submits the form

### ISSUE THREAD

**Issue by srerickson** [psu-libraries/scholarsphere#891]
Hitting enter when search for creators, submits the form (the page redirects to the work page).  

Note - this should be solved by moving help text to above the edit field. Screen reader user will know to arrow down for search results rather than hitting enter

**Comment by nicktiberi** [comment]
@srerickson - Is this essentially what you are looking for? I adjusted some of the margins to make things a bit cleaner with the tip now above the creator search box.

<img width="1194" alt="Screen Shot 2021-04-21 at 2 19 23 PM" src="https://user-images.githubusercontent.com/639920/115602329-d1087880-a2ac-11eb-9fe7-48124cfabd98.png">

**Comment by srerickson** [comment]
@nicktiberi, I'm having a hard time recalling what I meant when I wrote the "Note" part, so I can only interpret it literally. It looks like you've done what I described. The main concern is that the enter key shouldn't submit the whole form as it does right now.

Sorry for the poorly written issue.

**Comment by nicktiberi** [comment]
@srerickson - no worries! I'm guessing for now the enter key should just do nothing? Or maybe unfocus the search field?

We could also consider eventually tying it in with #945 and make the enter button add a new "custom" creator (the same action as selecting the "create your own" option in the dropdown.

**Comment by binkylush** [comment]
Accessibility testing recording - 
https://psu.mediaspace.kaltura.com/media/Binky+Lush%27s+Personal+Meeting+Room/1_frat7rs5

**Comment by awead** [comment]
## TODO

* hitting enter without making a selection defaults to the _first_ selection
    * this solves the "no results" scenario
* hitting enter without typing anything in the search box: nothing happens
* after making a selection, whether the "no results" or actual result, the cursor should change focus to Display Name
* once you're in the display name or any other creator field, hitting enter behaves as it currently does
